attestation-service/verifier/src/az_snp_vtpm/mod.rs

```diff
@@ -75,7 +80,11 @@ fn build_amd_chain() -> Result<AmdChain> {
     Ok(chain)
 }
 
-fn verify_snp_report(snp_report: &AttestationReport, vcek: &Vcek) -> Result<()> {
+fn verify_snp_report(
+    snp_report: &AttestationReport,
+    vcek: &Vcek,
+    init_data_hash: Option<&[u8]>,
+) -> Result<()> {
```

Comment:
Hi @fitzthum  Hi Tobin, I've changed the API of verifier of `SNP`. PTL if it looks good to you.

BTW, it will now change the behavior of current CoCoAS.